create completion from fine-tuned model

el-abdel opened the pull request.

el-abdel committed:
create completion from fine-tuned model

ceifa commented:
Thanks for the PR!

Instead of adding the model property to the `CompletionRequest` type, I think it should be better if the model was the first parameter of `completeFromModel`, just like the `complete` method. It because the model property should not be available at `complete`, and it's not optional for `completeFromModel`.

el-abdel committed:
[completeFromModel]: move model property to function params

el-abdel commented:
Hi,

I agree, even if `model ` property is not required in `CompletionRequest` but that could confuse users.
So i moved the prop to function params, also updated Read Me file adding the new function `completeFromModel`.
